protostar/commands/test/cheatcodes/cheatcode.py

```diff
@@ -0,0 +1,88 @@
+# pylint: disable=too-many-arguments
+# pylint: disable=too-many-instance-attributes
+from abc import abstractmethod
+import inspect
+from typing import TYPE_CHECKING, Type
+from starkware.cairo.lang.vm.relocatable import RelocatableValue
+from starkware.starknet.business_logic.execution.objects import (
+    TransactionExecutionContext,
+)
+
+from starkware.starknet.core.os.syscall_utils import BusinessLogicSysCallHandler
+from starkware.starknet.storage.starknet_storage import BusinessLogicStarknetStorage
+
+from protostar.commands.test.starkware.cheatable_carried_state import (
+    CheatableCarriedState,
+)
+from protostar.commands.test.starkware.cheatable_starknet_general_config import (
+    CheatableStarknetGeneralConfig,
+)
+
+if TYPE_CHECKING:
+    from protostar.commands.test.starkware.cheatable_execute_entry_point import (
+        CheatableExecuteEntryPoint,
+    )
+
+
+class Cheatcode(BusinessLogicSysCallHandler):
+    def __init__(
```

Comment:
In the future we can hoist this constructor to the parent class `CheatableSyscallHandler` and have `CheatedSyscallHandler` and `Cheatcode` inherit from it, but i don't want to pump this PR

Reply:
I think Cheatcode should inherit from `CheatableSysCallHandler`.